get-started-microservices-on-kubernetes/simple-microservice-llm/src/main/java/org/wildfly/examples/GettingStartedEndpoint.java

```diff
@@ -0,0 +1,51 @@
+package org.wildfly.examples;
+
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.UserMessage;
+import dev.langchain4j.memory.ChatMemory;
+import dev.langchain4j.memory.chat.MessageWindowChatMemory;
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.mistralai.MistralAiChatModel;
+import dev.langchain4j.model.mistralai.MistralAiChatModelName;
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+@Path("/")
+@RequestScoped
+public class GettingStartedEndpoint {
+
+    @Inject
+    @Named(value = "mistral")
```

Comment:
you should use ollama as the name so that Glow can provision your server for you